debian/control

```diff
@@ -26,9 +26,9 @@ Build-Depends: debhelper (>= 10~),
                libhiredis-dev,
                libriemann-client-dev (>= 1.6.0~),
                librabbitmq-dev,
-               python3, python3-dev, pep8, pylint3, dh-python,
+               python3, python3-dev, python3-pep8 | pep8, pylint3, dh-python,
```

Comment:
I'm not sure I understood what's going here, but if you added the python3-pep8 for backward-compatibility, I would suggest removing pep8, as pep8 is currently just a transitional package which does nothing else than depending on python3-pep8.

Reply:
This was originally converted from python2 equivalents, and Debian jessie didn't have a python-pep8 package (but it did have pep8). Now it does have python3-pep8, so this is indeed not needed.

Reply:
I am dropping this patch if we can merge #3 so I can rebase the rest of the patches on top of that.